dfsnew/dfs_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2015 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build unit integration
+
+package dfs_test
```

Comment:
This is another thing that would be nice to if we could all agree on one convention. Specifically, what package and file should contain the TestSuite definition when the package-under test has test cases scattered across several files?

I'm not objecting to the approach - just would like to see us move towards a more consistent approach (whatever that might be).